instrumentation/opentelemetry-instrumentation-redis/tests/test_redis.py

```diff
@@ -141,3 +141,60 @@ def request_hook(span, conn, args, kwargs):
 
         span = spans[0]
         self.assertEqual(span.attributes.get(custom_attribute_name), "GET")
+
+
+class TestRedisIntegrationMetric(TestBase):
+
+    def setUp(self):
+        super().setUp()
+        RedisInstrumentor().instrument(meter_provider=self.meter_provider)
+
+    def tearDown(self):
+        super().tearDown()
+        RedisInstrumentor().uninstrument()
+
+
+    @staticmethod
+    def redis_get_multiple_connections():
+        pool = redis.ConnectionPool(host='localhost', port=6379, db=0)
+        c1 = pool.get_connection('_')
+        c2 = pool.get_connection('_')
+        c3 = pool.get_connection('_')
+
+    @staticmethod
+    def redis_get():
+        pool = redis.ConnectionPool(host='localhost', port=6379, db=0)
+        redis_client = redis.Redis(connection_pool=pool)
+        redis_client.get('foo')
+
+
+    def test_basic_metric_success_redis(self):
+        self.redis_get()
+        expected_attributes = {
+            "db.client.connections.usage": 1,
+        }
+        for (
+            resource_metrics
+        ) in self.memory_metrics_reader.get_metrics_data().resource_metrics:
+            for scope_metrics in resource_metrics.scope_metrics:
+                for metric in scope_metrics.metrics:
+                    for data_point in metric.data.data_points:
+                        self.assertDictEqual(
+                            expected_attributes, dict(data_point.attributes)
+                        )
+
+    def test_multiple_connections_metric_success_redis(self):
+        self.redis_get_multiple_connections()
+
+        expected_attributes = {
+            "db.client.connections.usage": 3,
+        }
```

Comment:
This shouldn't be attribute. What you probably meant is the `value` of data point.